docs/experiment/sdks/ruby-sdk.md

```diff
@@ -0,0 +1,205 @@
+icon: material/language-ruby
+---
+
+Official documentation for Amplitude Experiment's server-side Ruby SDK implementation.
+
+[![Gem Version](https://badge.fury.io/rb/amplitude-experiment.svg)](https://badge.fury.io/rb/amplitude-experiment)
+
+!!!info "SDK Resources"
+     [:material-github: Github](https://github.com/amplitude/experiment-ruby-server) · [:material-code-tags-check: Releases](https://github.com/amplitude/experiment-ruby-server/releases) · [:material-book: API Reference](https://amplitude.github.io/experiment-ruby-server/)
+
+Ruby SDK supprts [remote](../general/evaluation/remote-evaluation.md) evaluation now.
+
+## Remote evaluation
+
+Implements fetching variants for a user via [remote evaluation](../general/evaluation/remote-evaluation.md).
+
+### Install
+
+!!!info "Ruby version compatibility"
+
+    The Ruby Server SDK works with Ruby 2.0+.
+
+Install the Ruby Server SDK with bundler or gem directly.
+
+=== "bundler"
+
+    ```bash
+    gem 'amplitude-experiment'
+    ```
+
+=== "gem"
+
+    ```bash
+    gem install amplitude-experiment --pre
+    ```
+
+!!!tip "Quick Start"
+
+    1. [Initialize the experiment client](#initialize)
+    2. [Fetch variants for the user](#fetch)
+    3. [Access a flag's variant](#fetch)
+
+    ```ruby
+    require 'amplitude-experiment'
+
+    # (1) Initialize the experiment client
+    experiment = Experiment.init('<DEPLOYMENT_KEY>', Experiment::Config.new(
+      fetch_timeout_millis: 500,
+      fetch_retries: 1,
+      fetch_retry_backoff_min_millis: 0,
+      fetch_retry_timeout_millis: 500
+    ))
+
+    # (2) Fetch variants for a user
+    user = Experiment::User.new(
+      user_id: 'user@company.com',
+      device_id: 'abcdefg',
+      user_properties: {
+        'premium' => true
+      }
+    )
+    variants = experiment.fetch(user)
+
+    # (3) Access a flag's variant
+    variant = variants['YOUR-FLAG-KEY']
+    unless variant.nil?
+        if variant.value == 'on'
+            # Flag is on
+        else
+            # Flag is off
+        end
+    end
+    ```
+
+### Initialize
+
+The SDK client should be initialized in your server on startup. The [deployment key](../general/data-model.md#deployments) argument passed into the `apiKey` parameter must live within the same project that you are sending analytics events to.
+
+```ruby
+init(api_key, config = nil) ⇒ Client
+```
+
+| Parameter | Requirement | Description |
+| --- | --- | --- |
+| `apiKey` | required | The [deployment key](../general/data-model.md#deployments) which authorizes fetch requests and determines which flags should be evaluated for the user. |
+| `config` | optional | The client [configuration](#configuration) used to customize SDK client behavior. |
+
+!!!info "Timeout & Retry Configuration"
+**The default timeout is too high for most server environments, and default retry configuration is 0**. Please configure the timeout and retry options to best fit your performance requirements.
```

Comment:
remove "The default timeout is too high for most server environments, and default retry configuration is 0"